sample_workloads/lit-gpt-demo/scripts/litgpt_container_entrypoint.sh

```diff
@@ -47,27 +47,30 @@ export OMP_NUM_THREADS=12
 set_nccl_specific_configuration() {
   if [[ "$USE_TCPX" == "yes" ]]; then
     echo "Using TCPX"
-    export NCCL_DEBUG_SUBSYS=INIT,GRAPH,ENV,TUNING,NET,VERSION
-    export NCCL_DEBUG=INFO
-    # export NCCL_DEBUG=WARN
-    export NCCL_SOCKET_IFNAME=eth0
     export NCCL_CROSS_NIC=0
     export NCCL_ALGO=Ring
     export NCCL_PROTO=Simple
-    export NCCL_NSOCKS_PERTHREAD=2
+    export NCCL_NET_GDR_LEVEL=PIX
+    export NCCL_P2P_PXN_LEVEL=0
+    export NCCL_DEBUG_SUBSYS=INIT,GRAPH,ENV,TUNING,NET,VERSION
+    export NCCL_DEBUG=INFO
```

Comment:
@Chris113113 what are your thoughts on the debugging flags, should we keep these in (lines 55-56).

Reply:
I think it's fine to keep them. It lets users see the logs by default.